src/pages/_app.page.js

```diff
@@ -39,14 +39,9 @@ const App = ({ Component, pageProps, router }) => {
     const { status, statusText, data } = err;
     const errorMsg = data?.detail || statusText;
 
-    if (status === 401) {
-      // TODO: jump to login page
-    } else if (status === 403) {
-      setErrorStatus(403);
-      setErrorMsg('很抱歉，您无权访问这个页面');
-    } else if (status === 404) {
-      setErrorStatus(404);
-      setErrorMsg('很抱歉，这个页面找不到了');
+    if ([403, 404].includes(status)) {
+      setErrorStatus(status);
+      setErrorMsg(errorMsg);
```

Comment:
Hey @634750802 , I think we should still let the back-end handle the error messages and translations as we decided. So I roll it back to the back-end solution. If errorMsg is not provided by back-end, it will fallback to the message mappings defined in https://github.com/pingcap/tug-website/pull/93/files#diff-74993e78989ef8753b11f3d8e863060ded659a36326fc1543be9dede295f297e